sql/modules/Roles.sql

```diff
@@ -195,16 +197,37 @@ GRANT SELECT ON location_class_to_entity_class TO PUBLIC;
 
 
 \echo BASE ROLES
-SELECT lsmb__create_role('base_user');
+SELECT lsmb__create_role(
+  'base_user',
+  $DOC$
+  Users need to be given this role in order to be granted access to the database schema which holds all LedgerSMB objects.
```

Comment:
Please add to base_user.

This role only allows access to menu items Preferences, Logout, and New Window.  The user basically cannot do anything without added additional roles.

Reply:
Done.